packages/sdk-core/src/model/report/BacktraceReport.ts

```diff
@@ -47,23 +57,29 @@ export class BacktraceReport {
         if (data instanceof Error) {
             this.annotations['error'] = data;
             this.classifiers = [data.name];
-            this.message = data.message;
-            this.stackTrace = data.stack ?? '';
+            const stackInfo = {
+                stack: data.stack ?? '',
+                message: data.message,
+            };
+            this.stackTrace['main'] = stackInfo;
```

Comment:
```suggestion
            this.stackTrace['main'] = {
                stack: data.stack ?? '',
                message: data.message,
            };
```

what do you think about this?

Reply:
I like it, fixed